src/main/java/org/opentripplanner/routing/graph/GraphIndex.java

```diff
@@ -441,7 +442,7 @@ protected boolean lessThan(TripTimeShort tripTimeShort, TripTimeShort t1) {
                 int sidx = 0;
                 for (Stop currStop : pattern.stopPattern.stops) {
                     if (currStop == stop) {
-                        if(pattern.stopPattern.pickups[sidx] == pattern.stopPattern.PICKDROP_NONE) continue;
+                        if(omitNonPickups && pattern.stopPattern.pickups[sidx] == pattern.stopPattern.PICKDROP_NONE) continue;
```

Comment:
No, we are talking about stops that  have dropoff, but pickups aren't allowed. In Oslo, this is the case with most region buses when approaching the urban area. They let off passengers on the way, but not taking any new ones on board.

So, the variable name describes a call that is not for pickup. I could rename it omitPickupOnly or something similar.